dali/pipeline/operators/crop/bbox_crop.cc

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/operators/crop/bbox_crop.h"
+
+namespace dali {
+
+DALI_SCHEMA(BBoxCrop)
+    .DocStr(
+        R"code(Perform a prospective crop to an image while keeping bounding boxes consistent. Inputs must be supplied as two Tensors:
+        `Images` containing image data in NHWC format, and `BBoxes` containing bounding boxes represented as `[l,t,r,b]`.
```

Comment:
Now it is fine but if we plan to use GPU crop for the images as well, it would be good to accept raw images and here extract their sizes or provide operator that will extract this sizes from raw images and supply us with appropriate tensor.
Let us leave it as it is but when GPU crop is ready I think it need to be reworked.

Reply:
Trackend internally: DALI-305